.changeset/sour-donkeys-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/select-utils': minor
+---
+
+add border bottom to the select input group header menu
```

Comment:
Since we are updating both the border and the font weight, perhaps we can be more generic here:

```suggestion
Small styles adjustments to dropdown menu groups headers.
```

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2690/commits/2e134ee172617b000a9bbca77df9a58706a2eb6c

Thank you.